docs/installationAndConfiguration.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Installation & Configuration
+nav_order: 2
+---
+
+Install this tool using the VS Code built-in [marketplace](https://marketplace.visualstudio.com/items?itemName=mlxprs.mlxprs). Search “MarkLogic” from the Extension tab of the activity bar. Click “Install” to download and install the extension.
+
+
+
+The MarkLogic extension may be configured in the VS Code Settings. You can also change the configuration settings just for the workspace by adding the values to the `settings.json` file in the .vscode directory of your workspace.
```

Comment:
I think this is worth explaining - I often forget how to actually get to the settings. The approach I just used was choosing "Preferences: Open User Settings" and then we should tell the user to look under "Extensions" and then "MLXPRS".